1-development/post-rating/src/main/java/com/invoicexpress/postrating/service/ScorePostDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.invoicexpress.postrating.service;
+
+import com.invoicexpress.postrating.entity.Post;
+
+
+public class ScorePostDTO {
```

Comment:
hasCode() and equals()? Do you see any advantage overriding those methods or not? And implements Comparable?

Reply:
Sometimes we need to override those methods when we need to check the objects based on some the property (business logic).
Yes, I could implement the comparable on this object and used to compare the score property.